apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -159,6 +160,10 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
           }
         />
       </div>
+      <hr className="border-subtle [&:has(+div:empty)]:hidden" />
```

Comment:
This hidden classname is on there for now because the `BookerLayoutSelector` can hide itself based on a feature toggle. In that case it would result in two lines right after each other, which is ugly. So we hide one for now.

Reply:
very clever